chessboard/index.css

```diff
@@ -0,0 +1,30 @@
+#chess-container{
```

Comment:
The CSS of the board can be improved, also adding labels to the chess board, like a,b,c,d etc, and 1,2,3,4, etc can be a good addition 
I guess since the page's background is white too, hence it's getting a bit difficult to observe the logic at first glance